variables.tf

```diff
@@ -284,7 +284,10 @@ variable "efs_volumes" {
 }
 
 variable "bind_mount_volumes" {
-  type = list(any)
+  type = list(object({
+    host_path = string
```

Comment:
Understood moving this list to an object makes `host_path` not optional. Are you guys adverse to adding the new optional object syntax introduced in 1.3.0 or is the plan to keep this pre 1.3.0 compliant?